### ISSUE TITLE
Can't run wp-cli when plugin activated

### ISSUE THREAD

**Issue by copyhold** [littlebizzy/force-https#6]
```
wp plugin deactivate autoptimize
Warning: Some code is trying to do a URL redirect. Backtrace:
#0  WP_CLI\Utils\wp_redirect_handler(https://) called at [/home/soundsystemsco/public_html/wp-includes/class-wp-hook.php:288]
#1  WP_Hook->apply_filters(https://, Array ([0] => https://,[1] => 301)) called at [/home/soundsystemsco/public_html/wp-includes/plugin.php:203]
#2  apply_filters(wp_redirect, https://, 301) called at [/home/soundsystemsco/public_html/wp-includes/pluggable.php:1199]
#3  wp_redirect(https://, 301) called at [/home/soundsystemsco/public_html/wp-content/plugins/force-https-littlebizzy/core/redirect.php:91]
#4  FHTTPS_Core_Redirect->redirect() called at [/home/soundsystemsco/public_html/wp-content/plugins/force-https-littlebizzy/core/redirect.php:68]
#5  FHTTPS_Core_Redirect->start() called at [/home/soundsystemsco/public_html/wp-includes/class-wp-hook.php:286]
#6  WP_Hook->apply_filters(, Array ([0] => )) called at [/home/soundsystemsco/public_html/wp-includes/class-wp-hook.php:310]
#7  WP_Hook->do_action(Array ([0] => )) called at [/home/soundsystemsco/public_html/wp-includes/plugin.php:453]
#8  do_action(plugins_loaded) called at [/home/soundsystemsco/public_html/wp-settings.php:327]
#9  require(/home/soundsystemsco/public_html/wp-settings.php) called at [phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php:1188]
#10 WP_CLI\Runner->load_wordpress() called at [phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php:1111]
#11 WP_CLI\Runner->start() called at [phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Bootstrap/LaunchRunner.php:23]
#12 WP_CLI\Bootstrap\LaunchRunner->process(WP_CLI\Bootstrap\BootstrapState Object ([WP_CLI\Bootstrap\BootstrapStatestate] => Array ())) called at [phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/bootstrap.php:74]
#13 WP_CLI\bootstrap() called at [phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/wp-cli.php:23]
#14 include(phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/wp-cli.php) called at [phar:///usr/local/bin/wp/php/boot-phar.php:11]
#15 include(phar:///usr/local/bin/wp/php/boot-phar.php) called at [/usr/local/bin/wp:4]
```

After I disable the forceHTTPS , wp-cli works properly.

**Comment by jessuppi** [comment]
Thanks for the report, we've pushed a temp version 1.2.1 with a patch:

https://github.com/littlebizzy/force-https/tree/1.2.1

However we will not release this version on WordPress.org as 1.3.0 will be released shortly.

**Comment by deltafactory** [comment]
Related to #2 that I had reported in July.

**Comment by jessuppi** [comment]
@deltafactory Apologies for the delay, thanks so much for your reports!

**Comment by jessuppi** [comment]
Fixed in version 1.3.0 thanks!

https://github.com/littlebizzy/force-https/tree/1.3.0
